products/statement-generator/src/components-layout/Affirmation.tsx

```diff
@@ -59,7 +60,9 @@ const Affirmation = () => {
         <p>{t(affirmationData.description)}</p>
       </div>
 
-      <div className={utilityClasses.buttonContainer}>
+      <div
+        className={`${utilityClasses.buttonContainer} ${utilityClasses.btnJustifyRight}`}
+      >
         <Button
           hasArrow
           className={utilityClasses.buttonRight}
```

Comment:
Aaah, also just noticed that this button already has a class passed in that's meant to justify it to the right `utilityClasses.buttonRight`. 

It uses `margin-left: auto`, which would typically work, except that it's being overriden by CSS from `.MuiButtonBase-root`. MUI injects their CSS last by default so their CSS wins out 🤷 

I'm ok with your solution in this scenario, but should remove the `utilityClasses.buttonRight` from this Button as it's not doing anything.

Reply:
I cleaned it up following your suggestion. Thank you for the explanation, I wasn't sure why that line of code wasn't being run, but your explanation provided me a lot of insight on MUI. Thanks!